### ISSUE TITLE
Readme contains Invalid nuget package installation instructions

### ISSUE THREAD

**Issue by kobruleht** [web-eid/web-eid-authtoken-validation-dotnet#19]
Tryin to use installation instructions by running in nuget console

`Add-Source -Name "Web eID GitLab" -Source "https://gitlab.com/api/v4/projects/35362906/packages/nuget/index.json"
`
Throws error

```
Add-Source : The term 'Add-Source' is not recognized as the name of a cmdlet, function, script file, or operable program. 
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ Add-Source -Name "Web eID GitLab" -Source "https://gitlab.com/api/v4/ ...
+ ~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (Add-Source:String) [], CommandNotFoundException
    + FullyQuali
```

**Comment by mrts** [comment]
Thanks for pointing this out! We will fix the instructions as soon as possible. Patches welcome!

**Comment by realmerx** [comment]
Quick fix until documentation is updated:

`dotnet nuget add source https://gitlab.com/api/v4/projects/35362906/packages/nuget/index.json --name "Web eID GitLab"`

**Comment by mrts** [comment]
Fixed with #33
